### PR TITLE
Database for staff

### PR DESCRIPTION
**Pull request by Lim-Shi-Han**

### CODE REVIEW

File: src/main/java/seedu/duke/manager/StaffManager.java [hunk 1]
```diff
@@ -103,4 +120,16 @@ public void deleteByStaffId(int staffId) {
     public int getNumOfStaffs() {
         return staffs.size();
     }
-}
+
+    @Override
+    protected void loadData() throws Exception {
+        this.staffs = new ArrayList<>();
+        ArrayList<?> list = (ArrayList<?>) this.load();
+        this.setStaffs((ArrayList<Staff>) list);
```
Comment (darylhjd) [inline src/main/java/seedu/duke/manager/StaffManager.java:128]: you will need to loop through the array list, then cast to the actual object (in this case, `Staff`) and add individually. This helps to make sure that an exception will be raised here and not anytime in the future if you want to access the list and suddenly the object is not of the correct type.

**Review by darylhjd — CHANGES_REQUESTED** [review]

File: src/main/java/seedu/duke/manager/StaffManager.java (hunk 1 above)
Comment (Lim-Shi-Han) [reply src/main/java/seedu/duke/manager/StaffManager.java:128]: Okay

File: src/main/java/seedu/duke/manager/StaffManager.java [hunk 2]
```diff
@@ -103,4 +120,18 @@ public void deleteByStaffId(int staffId) {
...
         return staffs.size();
     }
-}
+
+    @Override
+    protected void loadData() throws Exception {
+        this.staffs = new ArrayList<>();
+        ArrayList<?> list = (ArrayList<?>) this.load();
+        for (Object object : list) {
+            this.loadStaff((Staff) object);
```
Comment (darylhjd) [inline src/main/java/seedu/duke/manager/StaffManager.java:129]: technically dunnid a method for it ah can just this.staffs.add() or smth or ok

**Review by darylhjd — APPROVED** [review]

File: src/main/java/seedu/duke/manager/StaffManager.java (hunk 2 above)
Comment (Lim-Shi-Han) [reply src/main/java/seedu/duke/manager/StaffManager.java:129]: Ok ah that's true I might edit it in the next pr

**Pull request merged**